src/plugins/vis_builder/public/plugin.ts

```diff
@@ -120,7 +120,7 @@ export class WizardPlugin
       appExtensions: {
         visualizations: {
           docTypes: [PLUGIN_ID],
-          toListItem: ({ id, attributes }) => ({
+          toListItem: ({ id, attributes, updated_at }) => ({
```

Comment:
This is a known lint error. Since we cant do much to change the `updated_at` naming convention, alias this as `updated_at: updatedAt` here and reassign it on line 133 as `updated_at: updatedAt`. This should get around the lint error